apps/vscode.talon

```diff
@@ -46,9 +46,9 @@ console up: user.ide_terminal_scroll_up()
 console down: user.ide_terminal_scroll_down()
```

Comment:
Do we still need any of these commands now that we're using ide.talon? Seems to be a lot of overlap, this should probably be reserved for anything that's specific to VSCode.

Reply:
Agreed I think these are also legacy, I will attempt review  each and make changes via a  new PR as discussed